src/requests/verify-request.spec.ts

```diff
@@ -54,6 +67,36 @@ describe('isVerifiedRequest', () => {
     assert(verifyRequest(VALID_SECRET, incomingRequest, 0))
   })
 
+  it('verifies a verified request with user-id', () => {
+    const now = Date.now()
+    const incomingRequest = makeIncomingRequest(
+      {
+        headers: {
+          Authorization: 'Bearer TOKEN',
+        },
+      },
+      now,
+      { [ContentfulUserIdHeader]: 'my-user' }
```

Comment:
Segueing comments above, I would also test the case where a non-normalized header comes in